packages/template/package-template.json

```
@@ -84,7 +84,13 @@
     "//": "END_PLATFORM",
 
     "build": "rimraf dist && pnpm run codegen && tsdown",
-    "dev": "concurrently -n \"build,codegen\" -k \"tsdown --watch\" \"pnpm run codegen:watch\"",
+
+    "//": "IF_PLATFORM template react-like",
+    "dev": "concurrently -n \"build,env,css-tw,css-sc\" -k \"tsdown --watch\" \"chokidar --silent './scripts/generate-env.ts' -c 'pnpm run env' --throttle 2000\" \"tailwindcss -i ./src/global.css -o ./src/generated/tailwind.css --watch\" \"chokidar --silent './src/generated/tailwind.css' -c 'pnpm run css-sc' --throttle 2000\"",
```

Biome (2.5.1)

[error] The key dev was already declared.

(lint/suspicious/noDuplicateObjectKeys)